core/common/src/rpc/web/BentleyCloudRpcManager.ts

```diff
@@ -46,18 +48,34 @@ export abstract class BentleyCloudRpcConfiguration extends RpcConfiguration {
   public abstract override readonly protocol: BentleyCloudRpcProtocol;
 }
 
+/** Operating parameters for Bentley cloud RPC interface deployments.
+ * @beta This is only required to support accessing internal state returned from BentleyCloudRpcManager.
+ * Most use cases can be supported without accessing the internal members of this class.
+ */
+export class BentleyCloudRpcConfigurationHolder {
```

Comment:
@swbsi, if the other classes are all marked `beta` I don't think we need to hide the RpcConfiguration anymore and instead just use the existing class and refactor in 5.0.

Reply:
It’s purpose was to prevent the release level of BentleyCloudRpcManager from rippling through the rest of the api.

Reply:
I realize it doesn’t improve the internal status of RpcConfiguration et al.

Reply:
I don't really understand what you mean by rippling... could you provide a specific example? Regarding mixing beta with internal, you're already doing it in this class (beta class with internal properties), or did you mean something else?

Reply:
In that case I would just make `RpcConfiguration` beta, but leave all the properties internal. I would also change everything that was made public in this PR, to beta, because we're probably going to want to remove it all anyway (I'm just guessing here, maybe you have a different plan)

Reply:
Yes, that's basically what Caleb is suggesting. I'm going to try that...